x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -335,6 +335,36 @@ export const enrichExceptionItemsWithOS = (
   });
 };
 
+/**
+ * Returns given exceptionItems with all hash-related entries lowercased
+ */
+export const lowercaseHashValues = (
+  exceptionItems: Array<ExceptionListItemSchema | CreateExceptionListItemSchema>
+): Array<ExceptionListItemSchema | CreateExceptionListItemSchema> => {
+  return exceptionItems.map((item) => {
+    const newEntries = item.entries.map((itemEntry) => {
+      if (itemEntry.field.includes('.hash')) {
```

Comment:
This LGTM for merging, but you could be a little more specific here and check for `.hash.`. Can implement later if you think it's worth doing. Thanks for doing this!

Reply:
ECS has `user.hash` with no subfields unlike the other `hash` fields so we may want to stick with `.hash`

Reply:
Okies. As long as we don't have any other collisions here...